schema.ts

```diff
@@ -992,6 +992,14 @@ export interface IdentityDefinition {
   dynamicUrl?: string;
   /** The ID of another pack, if you are trying to reference a value from different pack. */
   packId?: number;
+  /**
+   * By default, result sets returned by dynamic sync tables will not be merged together for the purposes of presenting
```

Comment:
perhaps "merged together across different dynamicUrl values"?

Reply:
Yup, will update before merging. (Since I released already though won't cut another release for the correction.).